src/tests/get_item.c

```diff
@@ -77,12 +77,12 @@ int main(int argc, char *argv[])
 	}
```

Comment:
Actually, until `LXC 3.0` we should still run tests for the old configuration keys. So this file should only test for `lxc.network.{[i]}.*` related keys. The new network configuration keys are already tested in the `parse_config_file.c` test I added a while back.